core/src/components/checkbox/checkbox.scss

```diff
@@ -50,30 +60,90 @@ label {
   opacity: 0;
 }
 
+.checkbox-wrapper {
+  display: flex;
+
+  flex-grow: 1;
+
+  align-items: center;
+
+  height: 100%;
+  
+  cursor: inherit;
```

Comment:
Checkbox does not have `cursor: pointer` set on the host, so there's no indication that it is interactive. The legacy checkbox appears to place this on the invisible `label` element, but we may want to do this on the host so developers can easily override it.